warehouse/utils/utils.go

```diff
@@ -470,6 +481,19 @@ func GetConfigValueBoolString(key string, warehouse WarehouseT) string {
 	return "false"
 }
 
+func GetConfigValueAsMap(key string, warehouse WarehouseT) map[string]interface{} {
```

Comment:
Can we change the function signature to take destinationConfig instead of warehouse?

```
func GetConfigValueAsMap(key string, destConfig map[string]interface{} ) map[string]interface{} 
```